conda/recipes/pylibcugraph/meta.yaml

```
@@ -25,13 +25,16 @@ build:
 
 requirements:
   build:
+    - cmake {{ cmake_version }}
     - {{ compiler('c') }}
     - {{ compiler('cxx') }}
     - {{ compiler('cuda') }} {{ cuda_version }}
     - sysroot_{{ target_platform }} {{ sysroot_version }}
   host:
     - python x.x
     - cython>=0.29,<0.30
+    - cmake>=3.20.1,!=3.23.0
```

Same comment as before, check if necessary before committing my suggestion.
```suggestion
    - cmake {{ cmake_version }}
```